app/helpers/self_service/self_service_helper.rb

```diff
@@ -45,9 +46,12 @@ def office_address_table_cell(office)
         .join(', ')
     end
 
-    def render_onboarding_message(page)
-      render partial: "self_service/onboarding/#{page}",
-             locals: { principal: current_user.principal }
+    def render_onboarding_message
+      return if current_user.principal.onboarded?
+
+      content_for :onboarding_message do
+        render 'shared/onboarding_message'
+      end
```

Comment:
For clarity, if we move it inline we get extra spacing between the notifications. Although we could remove that with CSS, my concern is that we are then separating those notifications conceptually.

### Using `content_for`

![screenshot 2015-09-04 15 00 09](https://cloud.githubusercontent.com/assets/52965/9685372/b2d1b332-5315-11e5-8679-799b3ef47cf0.png)

### Using directly

![screenshot 2015-09-04 15 01 27](https://cloud.githubusercontent.com/assets/52965/9685404/ee57c9d2-5315-11e5-9dd6-349d761cebea.png)